pkg/store/forward/forward.go

```diff
@@ -0,0 +1,130 @@
+package forward
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/golang/snappy"
+	"github.com/openshift/telemeter/pkg/store"
+	clientmodel "github.com/prometheus/client_model/go"
+	"github.com/prometheus/prometheus/prompb"
+)
+
+const metricName = "__name__"
+
+type Store struct {
```

Comment:
rather than expose this type, let's make this unexported and return a `store.Store` from the `New` func. This way we can keep this tied to the store interface

Reply:
I tried to follow the practice of always returning am exported concrete type, even when composing multiple types using the same interface. But I can update this.